src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -1140,7 +1140,8 @@ void KaleidoScope_DrawInfoPanel(PlayState* play) {
                 gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 255, 255, 255, 255);
             }
 
-            POLY_OPA_DISP = KaleidoScope_QuadTextureIA4(POLY_OPA_DISP, pauseCtx->nameSegment, 128, 16, 0);
+            POLY_OPA_DISP = KaleidoScope_QuadTextureIA4(POLY_OPA_DISP, pauseCtx->nameSegment, ITEM_NAME_TEX_WIDTH,
+                                                        ITEM_NAME_TEX_HEIGHT, 0);
```

Comment:
these dimensions are also `MAP_NAME_TEX1_WIDTH`, `MAP_NAME_TEX1_HEIGHT` (`nameSegment+0` may hold either), worth a comment imo

Reply:
I don't really understand what the conditions for this code to run are. Can it run for both item_name and map_name textures, or is it intended for one or the other? If it's only intended for one I think whichever one it is should be used and it's not worth a comment (unless there's a bug)

Reply:
basically, this code runs most of the time, it's the label at the bottom of the pause menu that shows on every page (except when the cursor is on the "special positions" for switching page)

Don't be deceived by `cursorItem`/`namedItem`, it doesn't always hold items, it holds a map/area index when on the pause menu's world map page

Reply:
I see, thanks.
I've added some comments about this assumption the code makes